recursio-client/Global/InputManager.gd

```diff
@@ -1,6 +1,23 @@
 extends Node
 
 
+signal controller_changed(controller)
+
+# joypad type or "keyboard"
+var _current_controller := ""
```

Comment:
Why don't you use an enum here? Less error prone

Reply:
Because the Godot implementation returns a String for the controller type and I don't want to map it to custom types... but this should be a minor change and could be adjusted at any time